src/components/Cluster/ClusterDetail/NodePool.tsx

```diff
@@ -176,7 +178,7 @@ class NodePool extends Component<INodePoolProps, INodePoolsState> {
         <NameWrapperDiv
           style={{ gridColumn: isNameBeingEdited ? '2 / 9' : undefined }}
         >
-          <NPViewAndEditName
+          <NPViewAndEditNameStyled
             // @ts-ignore
```

Comment:
Is this needed?

Reply:
It looks like...

Reply:
Probably because the `ViewAndEditName` UI component, isn't typed 😢

Reply:
Yep